src/random.c

```diff
@@ -38,9 +38,12 @@ u32 RandomUniform(enum RandomTag tag, u32 lo, u32 hi);
 __attribute__((weak, alias("RandomWeightedArrayDefault")))
 u32 RandomWeightedArray(enum RandomTag tag, u32 sum, u32 n, const u8 *weights);
 
+__attribute__((weak, alias("RandomElementArrayDefault")))
+const void *RandomElementArray(enum RandomTag tag, const void *array, size_t size, size_t count);
+
 u32 RandomUniformDefault(enum RandomTag tag, u32 lo, u32 hi)
 {
-    return lo + (((hi - lo) * Random()) >> 16);
+    return lo + (((hi - lo + 1) * Random()) >> 16);
```

Comment:
This is a very serious bug I discovered when testing `RandomElementArrayDefault`—it meant that any calls to `RandomUniform` were exclusive of `hi` instead of inclusive.